lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -2821,6 +2821,17 @@ mapping:
           Details" option in the history. This option also exposes the command line to
           non-administrative users. Administrators can always see dataset paths.
 
+      allow_tool_source_display:
```

Comment:
Maybe `enable_tool_source_display` to be consistent with other options (enable_tool_recommendations, enable_markdown_export, enable_tool_tags)? Seems that enable is used more?